src/backend/distributed/citus--7.4-1--7.4-2.sql

```diff
@@ -0,0 +1,35 @@
+/* citus--7.4-1--7.4-2 */
```

Comment:
This is going to drive me to rewrite our whole upgrade script infrastructure :-|

Reply:
(Where _this_ == an inability to see the git history of objects we define in our upgrade scripts)